janitor/timeseries.py

```diff
@@ -0,0 +1,128 @@
+"""
+Time series-specific data testing and cleaning functions.
+"""
+
+import pandas as pd
+import pandas_flavor as pf
+from pandas.api.types import is_datetime64_any_dtype as is_datetime
+from collections import namedtuple
+from janitor import check
+
+
+def _flag_missing_timestamps(
+    df: pd.DataFrame,
+    frequency: str,
+    column_name: str,
+    first_time_stamp: pd.Timestamp,
+    last_time_stamp: pd.Timestamp,
+) -> namedtuple:
+    """
+    Test if timestamps are missing
+
+    Utility function to test if input data frame is missing any timestamps relative to expected timestamps.
+    They are generated based on the first_time_stamp, last_time_stamp and frequency.
```

Comment:
One suggestion for a semantic line-breaks-compatible docstring

```suggestion
    Utility function to test if input data frame
    is missing any timestamps relative to expected timestamps.
    They are generated based on the first_time_stamp,
    last_time_stamp
    and frequency.
```